dhis-2/dhis-api/src/main/java/org/hisp/dhis/datavalue/DataExportParams.java

```diff
@@ -378,6 +403,28 @@ public DataExportParams setOrganisationUnits( Set<OrganisationUnit> organisation
         return this;
     }
 
+    public Set<OrganisationUnit> getOrgUnitParents()
```

Comment:
I would prefer not to have a specific "orgUnitParents" collection, and instead just have the existing org unit collection and in addition include an "org unit selection mode" property.

We have an `OrganisationUnitSelectionMode` enum in the system, where we have relevant values like `SELECTED` and `DESCENDANTS`. `SELECTED` is default and current behavior. `DESCENDANTS` means include the sub-hierarchy. This is more flexible as it allows for more modes/options.

See event analytics as an example, scroll to "ouMode":
 
https://docs.dhis2.org/master/en/dhis2_developer_manual/web-api.html#event-analytics

Sounds reasonable?

Reply:
Makes sense. Done!